.ci/publish-website

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Datawire. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+CHANGED_DOCS_FILES=$(git diff --name-only "$TRAVIS_COMMIT_RANGE" docs)
+if [ -z "$CHANGED_DOCS_FILES" ]; then
+    echo "No changes have been made to /docs/, website will not be updated..."
+    exit 0
+else
+  echo "Publishing changes to website..."
+fi
```

Comment:
This skips publishing website when no changes have been made to /docs/ directory.